### DISCUSSION TITLE
Welcome to polyRAD Discussions!

### DISCUSSION THREAD

**Discussion by lvclark** [lvclark/polyRAD#15] [General]
## 👋 Welcome!
 If you have questions about polyRAD or want help with errors, feel free to start a Discussion here!  You may also post to Issues if you suspect there is a problem with the software that needs to be fixed, or if there is an enhancement that you would like to see.  Don't worry about it too much, as I can easily convert an Issue to a Discussion and vice versa.

**Comment by heatherturtle** [comment]
Hi Lindsay, 
Can polyRAD be used for a smaller population of 60 individuals, or is it more suitable for larger populations?
